test/integration/test_api.py

```diff
@@ -408,6 +408,60 @@ async def test_validate_dictionary_choice(async_client, dictionary, filename, ex
     assert response.json()['data'][0]['dictionary'] == expected
 
 
+@pytest.mark.parametrize('response_type, response_type_result', [
+    ('inline', 'inline'),
+    ('attachment', 'attachment'),
+    (None, 'inline')  # Defaults to 'inline'
+])
+def test_get_ags_log(client, response_type, response_type_result):
+    """
+    Confirm that the endpoint can return the expected .pdf.
+    """
+    # Arrange
+    # Define the borehole ID to use for the test
+    bgs_loca_id = 20190430093402523419
+    query = f'/ags_log/?bgs_loca_id={bgs_loca_id}'
+    if response_type:
+        query += f'&response_type={response_type}'
+
+    # Act
+    with client as ac:
+        response = ac.get(query)
+
+    # Assert
+    # Check that the response status code is 200
+    assert response.status_code == 200
+    # Check that the response headers include the correct Content-Disposition header
+    content_disposition = f'{response_type_result}; filename="{bgs_loca_id}_log.pdf"'
+    assert response.headers["Content-Disposition"] == content_disposition
+    # Check that the response media type is "application/pdf"
+    assert response.headers["Content-Type"] == "application/pdf"
+    # Check that the response content is not empty
+    assert len(response.content) > 0
+    # Check it is a PDF file
+    assert response.content.startswith(b'%PDF')
+
+
+def test_get_unknown_ags_log(client):
+    """
+    Confirm that the endpoint can return the expected .pdf.
```

Comment:
Darn, this comment needs updating!

Reply:
https://github.com/BritishGeologicalSurvey/pyagsapi/pull/101/commits/7d482295a5467af05523265942ac9c1164ee13c1 fixed